### PR TITLE
Allow plugins to be loaded from /code/node_modules

### PR DESCRIPTION
**Pull request by GordonDiggs**
/cc @codeclimate/review 

This fixes the recurring bugsnag as well as #10

### CODE REVIEW

**Comment by pbrisbin** [comment]
Doesn't this just fix the specific plugin? Don't we need to support the user having their own plugins in `./node_modules`?

**Comment by GordonDiggs** [comment]
@pbrisbin ready for another look

File: Dockerfile [hunk 1]
```diff
@@ -10,4 +10,6 @@ USER app
 
 COPY . /usr/src/app
 
+ENV NODE_PATH /code/node_modules
```
Comment (pbrisbin) [inline Dockerfile]: Should this be something like `/code/node_modules:/usr/src/app/node_modules`?

Comment (pbrisbin) [inline Dockerfile]: and even `:/whatever/the/system/node_modules` too?

Comment (GordonDiggs) [inline Dockerfile]: Nope - by default it is nothing and is only used when the module can't be found elsewhere:

> If the NODE_PATH environment variable is set to a colon-delimited list of absolute paths, then node will search those paths for modules if they are not found elsewhere.

*- https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders*

Comment (pbrisbin) [inline Dockerfile]: Cool!

**Comment by pbrisbin** [comment]
LGTM

**Pull request merged**